README.md

```diff
@@ -20,6 +20,10 @@
 - [Chapter 16 - Refactoring SerialDate](#chapter15)
 - [Chapter 17 - Smells and Heuristics](#chapter17)
 
+<h1>Credits</h1>
+
+This document is a summary of the wonderful 2008 book **Clean Code: A Handbook of Agile Software Craftsmanship** by Robert C. Martin. [link](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882)
```

Comment:
```suggestion
This document is a summary of the 2008 book **Clean Code: A Handbook of Agile Software Craftsmanship** by Robert C. Martin. [link](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882)
```

Reply:
Thank you for flagging it. I wrote my notes as personal notes 8 Years ago. I never thought this repository would be relevant for someone else 😁

I left a minor request before merge it.